app/views/documents/index.html.erb

```diff
@@ -61,60 +61,101 @@
-          <% end %>
-        </tbody>
+  <div class="govuk-grid-row govuk archived-documents">
+    <h2 class="govuk-heading-m govuk-!-padding-left-3 govuk-!-padding-top-3 govuk-!-padding-bottom-3">
+      Archived documents
+    </h2>
+   <div class="govuk-grid-row govuk-!-padding-left-7 govuk-!-padding-right-7">
+    <% if filter_archived(@documents).present? %>
+        <div class="govuk-grid-column-one-quarter">
+          <p class="govuk-body govuk">
+            <strong>Document name</strong>
+          </p>
+        </div>
+        <div class="govuk-grid-column-one-quarter">
+          <p class="govuk-body">
+            <strong>Tags</strong>
+          </p>
+        </div>
+        <div class="govuk-grid-column-one-quarter">
+          <p class="govuk-body govuk-!-margin-bottom-1">
+            <strong>Date uploaded</strong>
+          </p>
+        </div>
+        <div class="govuk-grid-column-one-quarter">
+          <p class="govuk-body">
+            <strong>Officer's comment</strong>
+          </p>
+        </div>
+      </div>
+    <% else %>
+      <div class="govuk-grid-column-one-quarter">
+        <p class="govuk-body">
+          No documents archived
+        </p>
+      </div>
+    <% end %>
+  </div>
+  <% if filter_archived(@documents).present? %>
+    <ul class="app-task-list archived-documents">
+      <% filter_archived(@documents).each do |document| %>
+        <li class="app-task-list__item govuk-!-padding-6">
+          <div class="govuk-grid-column-one-quarter">
+            <p class="govuk-body govuk-!-margin-bottom-1">
+              <%= link_to image_tag(document.file.representation(resize: "150x100"), class: "govuk-!-width-full"),
+                          url_for_document(document), target: :_blank %>
+              <%= link_to document.name, url_for_document(document), target: :_new %>
+            </p>
+          </div>
+          <div class="govuk-grid-column-one-quarter">
+            <% if document.tags.present? %>
+              <% document.tags.each do |tag| %>
+                <strong class="govuk-tag govuk-tag--turquoise"><%= tag %></strong>
+              <% end %>
+            <% else %>
+              <p class="govuk-body govuk govuk-!-margin-bottom-1">
+                <em>No tags added</em>
+              </p>
+            <% end %>
+          </div>
+          <div class="govuk-grid-column-one-quarter">
+            <p class="govuk-body govuk govuk-!-margin-bottom-1">
+              <%= document.created_at.strftime("%e %b %Y") %>
+            </p>
+          </div>
+          <div class="govuk-grid-column-one-quarter">
+            <p class="govuk-body govuk govuk-!-margin-bottom-1">
+              <% if document.archive_reason.present? %>
+                <%= document.archive_reason %>
+              <% else %>
+                <em>No reason provided</em>
+              <% end %>
+            </p>
+          </div>
+        </li>
       <% end %>
-    </table>
+      </ul>
+    <% end %>
```

Comment:
I'm finding this quite hard to read! There are two `if/else` statements that are checking the same thing? It's tabular data that's not in a table?

Reply:
I've rationalised this so we don't have the same conditional. Essentially, we needed the whole page to align and look like current documents and archived documents are structured in the same way, so we either needed both sections to be a table or both to be a grid to ensure the lines and headings lined up, especially after we added in the thumbnails in the archive section, so I've left it as a grid.

Reply:
I'm splitting hairs a bit here, but would a table with custom column widths as described in https://design-system.service.gov.uk/components/table/ be an easier way of achieving this?